### DISCUSSION TITLE
Offcanvas navbar that expands into a normal navbar at a specific breakpoint bug

### DISCUSSION THREAD

**Discussion by Dreamtags** [twbs/bootstrap#38304] [Q&A]
There is a small bug if I use the code below to make my off canvas menu for mobile. When I have my screen somewhere between 1380 and 1400px and click on the hamburger menu to op the off canvas, it only shows the transparent background. I checked it and it also happens on the bootstrap site itself.

```html
<nav class="navbar navbar-expand-xxl bg-body-tertiary fixed-top">
  <a class="navbar-brand" href="#">Offcanvas navbar</a>
  <button class="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#navbarOffcanvasLg" aria-controls="navbarOffcanvasLg">
    <span class="navbar-toggler-icon"></span>
  </button>
  <div class="offcanvas offcanvas-end" tabindex="-1" id="navbarOffcanvasLg" aria-labelledby="navbarOffcanvasLgLabel">
    ...
  </div>
</nav>
```

**Comment by julien-deramond** [comment]
With which browser/OS combination? I haven't managed to reproduce it on my side. Maybe others succeeded?

**Reply by Dreamtags** [reply]
IOS 13.2.1 IMAC, Safari 16.3 and seems to work fine on chrome.

**Reply by julien-deramond** [reply]
I've created [this CodePen](https://codepen.io/julien-deramond/pen/mdGQROY) to try to reproduce it. But can't on Ventura 13.2.1 with Safari 16.3. Can you confirm that it happens with this CodePen too on your side?
If not, would it be possible to fork the CodePen and modify it so that I can actually see the issue?
If we manage to reproduce it, we could transform this discussion into an actual issue in the repo to treat it :)

**Reply by Dreamtags** [reply]
In codepen I cannot reproduce it. Even if I copy the code with the menu. If I add navbar-expand-xxl in the inspect element on the bootstrap off canvas navbar:
https://getbootstrap.com/docs/5.3/components/navbar/

There I can reproduce it.

**Reply by julien-deramond** [reply]
I've tried to modify `.offcanvas-lg` to `.offcanvas` dynamically in Chrome and Firefox. In Firefox, I can observe a white background for the offcanvas. This is probably what you've observed. But it doesn't happen when it is not modified dynamically. I don't think there's an issue here unless we can create a reproducible example. Offcanvas being a JS component, maybe modifying a class dynamically breaks something but IDK where exactly that could happen.

**Reply by Dreamtags** [reply]
what I get in this small range is when I click I get a transparent background only.
